Add hint to create Windows homedir symlink in WSL

victorlin opened the pull request:
### Description of proposed changes

This will help users who may be confused on how to access Windows files
from WSL.

### Related issue(s)

Closes #114.

### Testing

_N/A_

emmahodcroft commented:
I think this looks good - I do think adding such a shortcut is really, really helpful to Windows users. My only suggestion is to add something like, "If your computer's language is set to something other than English, be sure to change the folder names in the path to correspond. For example, on a German-language machine, the path may be `/mnt/c/Benutzer/<user>`."

I've run into this a number of times on international tutorials!

victorlin commented:
@emmahodcroft good point, I forgot about that discussion in the linked issue. Added your suggestion, and also:

1. Renamed `windows_user` to `windows_home`.
2. Use the `-w` option to warn if the source directory does not exist (no simple way to error, so the link would still be created but a warning should help).

j23414 reviewed (approved):
Happy to see the symlink! :D

emmahodcroft commented:
Looks great now @victorlin , thanks for including that!

tsibley commented on src/install.rst:
```diff
@@ -225,6 +225,19 @@ These instructions will install the Nextstrain CLI and tools to run and view you
+
+         .. code-block:: bash
+
+               ln -ws /mnt/c/Users/<user> ~/windows_home
```
Does WSL expose the typical Windows environment variables like `USERPROFILE` or `HOMEDRIVE` and `HOMEPATH`? If so, those could be used in this command to avoid the manual "replace X with Y" steps.

victorlin replied:
Oh good thinking! [From StackExchange](https://superuser.com/a/1568668):

```sh
wslpath "$(wslvar USERPROFILE)"
```

I'll make another PR using this.
